patientsearch/src/js/components/TimeoutModal.js

```diff
@@ -121,7 +121,7 @@ export default function TimeoutModal() {
     clearExpiredIntervalId();
     sessionStorage.clear();
     setTimeout(() => {
-      window.location = "/logout";
+      window.location = "/logout?timeout=true&user_initiated=true";
```

Comment:
`user_initiated`, at least for me, implies the user clicks "log out"? is that what's happening here?

Reply:
@ivan-c You are right, sorry, for the confusion.  `user_initiated` will only be sent as a parameter if user clicks on the logout button.  I did just fix it.